docs/content/misc/contribute.ngdoc

```diff
@@ -70,7 +71,7 @@ cd angular.js
 # Add the main AngularJS repository as an upstream remote to your repository:
 git remote add upstream "https://github.com/angular/angular.js.git"
 
-# Install node.js dependencies:
+# Install node.js dependencies(Note: Please check for the stable version of node before installing the essential packages):
```

Comment:
What is meant by "check for the stable version of node"? That you should make sure you are running the stable version of node?